content/support-matrix/delfin/dell-emc/vmax/_index.md

```diff
@@ -0,0 +1,448 @@
+            <td rowspan=9>Controllers</td>
+            <td>id</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>name</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>storage_id</td>
+            <td></td>
+        </tr>
+        <tr>
+           <td>native_controller_id</td>
+           <td></td>
+        </tr>
+        <tr>
+           <td>status</td>
+           <td></td>
+        </tr>
+        <tr>
+        <td>location</td>
+        <td></td>
+        </tr>
+        <tr>
+           <td>soft_version</td>
+           <td></td>
+        </tr>
+        <tr>
+           <td>cpu_info</td>
+           <td></td>
+        </tr>
+        <tr>
+           <td>memory_size</td>
+           <td></td>
+        </tr>
+        <tr>
+        <td rowspan="15">disk</td>
+        <td>id</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>name</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>storage_id</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>native_disk_id</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>serial_number</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>manufacturer</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>model</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>firmware</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>speed</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>capacity</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>status</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>physical_type</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>logical_type</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>health_score</td>
+        <td></td>
+        </tr>
+        <tr>
+        <td>native_disk_group_id</td>
+        <td></td>
+        </tr>
+     </tbody>
+ </table>
+ 
+ 
+ ### Alert data matrix
+ 
+ <table>
+     <thead>
+         <tr>
+             <th>Attribute</th>
+             <th>Additional information</th>
+         </tr>
+     </thead>
+     <tbody>
+         <tr>
+            <td>SYMAPI_AEVENT2_UID_EVT_RESTARTED</td>
```

Comment:
This informations are wrong.. alert fileds means the fields available in delfin for VMAX alerts . you can probably mention the curresponding VMAX event field if applicable check with @sushanthakumar for more details 
![image](https://user-images.githubusercontent.com/45681499/108025047-c41c9e00-704b-11eb-9814-dcd7603cb13c.png)

Reply:
done